src/static/js/modules/clear-form-buttons.js

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   Clear form button
+   - Clear checkboxes and selects
+   - Clear Chosen.js elements
+   - Clear jquery.custom-input elements
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+  $( '.js-form_clear' ).on( 'click', function() {
+    var $this = $( this ),
+        $form = $this.parents( 'form' );
```

Comment:
This is what we ended up with:

> Always declare variables in a consistent manner, and at the top of their scope. _Keeping variable declarations to one per line is encouraged._

These should be moved to single line declarations, but all these JS files need further attention. Let's fix the standards/lint issues in another PR after they are moved so it's easier to track the changes.